mantidproject/mantid
# Updated reduction packaging to include wavelength ranges

This PR allows a range set of wavelength ranges to be selected in the new ISIS SANS GUI. A reduction will then be done between each of these wavelength ranges with the resulting workspaces being output to the ADS in a group. 

**To test:**

- Get the sample data from here
[loqdemo.zip](https://github.com/mantidproject/mantid/files/1961183/loqdemo.zip)

- In the new ISIS sans GUI
- Load maskfile.txt as the user file
- Load load batch_mode_reduction.csv as the batch file.
- In the bottom right ensure that Use Optimizations is unticked.
- Go to Settings-> Q,Wavelength, Detector Limits
- In the wavelength group box change the step type to RangeLin
- Change the ranges box to 1:1:3
- Go back to the runs tab and click process.
- At the end the ADS should contain 3 group workspaces.
   - first_time_merged should contain 3 workspaces 
   - second_time_merged should contain 3 workspaces
   - LAB_and_HAB_workspaces_from_merged_reduction should contain 12 workspaces
- Check that the names all make sense



Fixes #21734 

. 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
